### DISCUSSION TITLE
Make scroll area with changing width keep its relative vertical scroll position

### DISCUSSION THREAD

**Discussion by Rusty-Cube** [emilk/egui#5261] [Q&A]
I have a `ScrollArea` that contains a `Grid` with many rows. I'm using `ScrollArea::show_rows` to display it efficiently. However, the `ScrollArea` is inside a side panel, that I want to have dynamic width. If I resize the side panel, the scroll bar seems to keep its absolute scroll position and the content flies by. It would be much more useful, if it would keep its relative scroll position, so the content stays in place. How do I do that?

https://github.com/user-attachments/assets/85042bb0-b0ad-4747-813a-e88527425150

**Comment by BeastLe9enD** [comment]
good question, I also have the same problem. My approach is manually scaling it, but I get other problems there. does someone have any ideas?
